recipes/highs/all/patches/0001-missing-includes-msvc15-1.4.2.patch

```diff
@@ -0,0 +1,48 @@
+diff --git a/extern/filereaderlp/reader.cpp b/extern/filereaderlp/reader.cpp
+index 86151120..71f6747a 100644
+--- a/extern/filereaderlp/reader.cpp
++++ b/extern/filereaderlp/reader.cpp
+@@ -2,6 +2,7 @@
+ 
+ #include "builder.hpp"
+ 
++#include <cctype>
+ #include <cstring>
+ #include <cassert>
+ #include <iostream>
+diff --git a/src/lp_data/HighsOptions.cpp b/src/lp_data/HighsOptions.cpp
+index 903a7c71..6e66e78e 100644
+--- a/src/lp_data/HighsOptions.cpp
++++ b/src/lp_data/HighsOptions.cpp
+@@ -17,6 +17,7 @@
+ 
+ #include <algorithm>
+ #include <cassert>
++#include <cctype>
+ 
+ // void setLogOptions();
+ 
+diff --git a/src/presolve/ICrash.cpp b/src/presolve/ICrash.cpp
+index 12a6fc3f..e7bf86a4 100644
+--- a/src/presolve/ICrash.cpp
++++ b/src/presolve/ICrash.cpp
+@@ -14,6 +14,7 @@
+ #include "presolve/ICrash.h"
+ 
+ #include <algorithm>
++#include <cctype>
```

Comment:
You should submit a pr upstream and then link to it in the conandata patch fields :)

Reply:
PR created and conandata updated